pkg/activator/activator.go

```diff
@@ -48,6 +49,14 @@ func (rev RevisionID) String() string {
 	return fmt.Sprintf("%s/%s", rev.Namespace, rev.Name)
 }
 
+// EndpointGetter is a functor that given namespace and name will
+// return the number of endpoints in the endpoinst resource or an error.
+type EndpointGetter func(string, string) (int32, error)
```

Comment:
Should all of the passed through functions get a type? Also: Should this be `EdnpointsCounter`? Or `EndpointCountGetter`?

Reply:
+1 also nit: it's pluralized `EndpointsCountGetter`  (a bit of an oddball 🙄 )

Reply:
Eh, I didn't like the other two, so I didn't grant them, but sure.